### ISSUE TITLE
Fit the PSF with least squares rather than EM.

### ISSUE THREAD

**Issue by rgiordan** [jeff-regier/Celeste.jl#42]

**Comment by jeff-regier** [comment]
What happened with EM?

**Comment by rgiordan** [comment]
See the thread on the inference mailing list.  EM is minimizing log loss, when what we really care about is squared error.

**Comment by jeff-regier** [comment]
OK to close this? (Is this something included in SloanDigitalSkySurvey.jl?)

**Comment by rgiordan** [comment]
Not until a new issue is opened on the sdss package at least...

**Comment by rgiordan** [comment]
Moved to https://github.com/rgiordan/SloanDigitalSkySurvey.jl/issues/3
